collect_app/src/main/java/org/odk/collect/android/preferences/dialogs/SavedFormListProgressDialog.kt

```diff
@@ -0,0 +1,14 @@
+package org.odk.collect.android.preferences.dialogs
+
+import android.content.Context
+import org.odk.collect.android.R
+import org.odk.collect.material.MaterialProgressDialogFragment
+
+class SavedFormListProgressDialog : MaterialProgressDialogFragment() {
```

Comment:
The idea behind https://github.com/getodk/collect/pull/5002 was to allow creating progress dialogs without subclasses but you ended up with a bunch of new ones. Please handle that according to https://github.com/getodk/collect/pull/5002

Reply:
Done.